values-schema.yaml

```diff
@@ -2579,7 +2579,7 @@ properties:
           - password
       hasCloudLB:
         default: false
-        description: 'Set this to true when an external LB exists or needs to be started (AWS ALB, Azure AppGW, Google Apigee). This will then be configured through ingress controllers. Expects existing LBs to terminate https. Currently this is only working correctly for Azure, and not for AWS and Google. AWS is close to completion.'
+        description: Set this to true when using a cloud specific ingress controller. By default the Nginx Ingress Controller is used.
```

Comment:
```suggestion
        description: Set this to true when an external (cloud) loadbalancer is in charge of terminating TLS and forwards traffic unencrypted. Otherwise Otomi's Nginx controller will create one.
```

Reply:
Thanks for suggestion. I rephrased it again.